spec/rubocop/config_validator_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::ConfigValidator, :isolated_environment do
+  include FileHelper
+
+  # before do
+  #   allow(default_config).to receive(:internal?).and_return(true)
+  # end
```

Comment:
Did you forget to remove this?

Reply:
Yes sorry this was something else I was working on and shouldn't have been checked in 🤦‍♂️

Reply:
Removed now, thanks @koic!